automated install/basic-install.sh

```diff
@@ -1434,99 +1995,139 @@ main() {
     # Clone/Update the repos
     clone_or_update_repos
 
-       # Install packages used by the Pi-hole
-    if [[ ${INSTALL_WEB} == true ]]; then
+    # Install packages used by the Pi-hole
+    if [[ "${INSTALL_WEB}" == true ]]; then
+      # Install the Web dependencies
       DEPS=("${PIHOLE_DEPS[@]}" "${PIHOLE_WEB_DEPS[@]}")
+    # Otherwise,
     else
+      # just install the Core dependencies
       DEPS=("${PIHOLE_DEPS[@]}")
     fi
     install_dependent_packages DEPS[@]
 
+    if [[ -x "$(command -v systemctl)" ]]; then
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(systemctl is-enabled lighttpd | grep -c 'enabled' || true)
+    else
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(service lighttpd status | awk '/Loaded:/ {print $0}' | grep -c 'enabled' || true)
+    fi
```

Comment:
On a clean install: 
`LIGHTTPD_ENABLED` defaults to `0` due to the service being disabled and stopped by default.
This results in `lighttpd` not being enabled and started upon a clean install.